packages/sirius-web/backend/sirius-web-domain/src/main/java/org/eclipse/sirius/web/domain/boundedcontexts/projectsemanticdata/ProjectSemanticData.java

```diff
@@ -43,7 +43,7 @@ public class ProjectSemanticData extends AbstractValidatingAggregateRoot<Project
     private UUID id;
 
     @Column("project_id")
-    private AggregateReference<Project, String> project;
+    private String project;
```

Comment:
Why this change? We are using the type AggregateReference to specifically know the type of identifier that is being used.